packages/components/select/src/select.vue

```diff
@@ -3,7 +3,7 @@
     ref="selectWrapper"
     v-click-outside:[popperPaneRef]="handleClose"
     :class="wrapperKls"
-    @click.stop="toggleMenu"
+    @click.stop="(e) => toggleMenu(e)"
```

Comment:
The event will be passed by default. Maybe we can revert this change.

Reply:
ok.